zchunk_format.txt

```diff
@@ -116,13 +121,14 @@ The index:
 +===============================+
 
 (Chunk stream will only exist if flag 0 is set to 1)
-[+===================+================+===================+
-[| Chunk stream (ci) | Chunk checksum | Chunk length (ci) |
-[+===================+================+===================+
+EXPERIMENTAL: (Uncompressed chunk checksum will only exist if flag 2 is set to 1)
+[+===================+================+=============================+
+[| Chunk stream (ci) | Chunk checksum | Uncompressed chunk checksum |
+[+===================+================+=============================+
```

Comment:
So this means that flag=1 is not supported anymore and replaced by Flag=2 (with all infos) ?

Reply:
No, the chunk stream is only ever used if flag 1 is set.  All I've done is added Uncompressed chunk checksum which is only ever used if flag 2 is set.  Because the flags are represented as bits, you can set flag 1 and flag 2 independently of each other.  So you can have the uncompressed chunk checksum without having the chunk stream.

I should probably figure out a better way of expressing this in the document.

Reply:
Let's see if my latest push makes it any clearer

Reply:
I've got a good chunk of spare time over the next couple of weeks.  If you're happy with how this looks, I'll see if I can get it done before the new year.